app/src/main/index.js

```diff
@@ -139,6 +153,8 @@ function createWindow() {
   mainWindow.once(`ready-to-show`, () => {
     setTimeout(() => {
       mainWindow.show()
+      showCreateAccountWindow(mainWindow)
```

Comment:
should probably be created as a result of the main UI starting an account creation. but I guess this was not intended here in terms of it being a spike.

Reply:
Correct.